node/forest_libp2p/src/behaviour.rs

```diff
@@ -56,7 +56,7 @@ use tiny_cid::Cid as Cid2;
 
 lazy_static! {
     static ref VERSION: &'static str = env!("CARGO_PKG_VERSION");
```

Comment:
This should use `FOREST_VERSION`

https://github.com/ChainSafe/forest/blob/main/forest/build.rs#L22

And we can remove `CURRENT_COMMIT`

Reply:
Thank you for this interesting reference.

Actually in the Image Build Test  the `FOREST_VERSION` is documented as "`v0.2.2/unstable/mainnet/`"

So, I was wondering why the build process did not crash on the `version()` function as expected according to:
[_Git_ Command Call](https://github.com/ChainSafe/forest/blob/e66d0f9e2f810752bd3330d9ae2370f7197b234b/forest/build.rs#L27-L30)
```
    let git_cmd = Command::new("git")
        .args(&["rev-parse", "--short", "HEAD"])
        .output()
        .expect("Git references should be available on a build system");
```
And I think the answer is that it only causes an error if it is unable to launch the child process 

So the child process is launched but it produces an error message that is propagated on the `STDERR` and returns an Error `Status Code` `127` like in:
```
$ docker exec -e FULLNODE_API_INFO="/ip4/127.0.0.1/tcp/1234/http" -it dda58b640601 bash
root@dda58b640601:/# git rev-parse --short HEAD ; echo "res: '$?'"
bash: git: command not found
res: '127'
root@dda58b640601:/# 
root@dda58b640601:/# git rev-parse --short HEAD 2>/dev/null ; echo "res: '$?'"
res: '127'
```
So, if the build is suppose to stop without git the `git_version!()` macro will actually produce the expected Error.

On the other hand the `forest/build.rs` application also could be the moment where the required `CURRENT_COMMIT` would be populated.
Or where a supposed "_release version file_" could be created and read.

Reply:
> This should use `FOREST_VERSION`
> 
> https://github.com/ChainSafe/forest/blob/main/forest/build.rs#L22
> 
> And we can remove `CURRENT_COMMIT`

Yes, it turns out that `CURRENT_COMMIT` is a local variable that is not used according to the study documented at:
[unused warning for `CURRENT_COMMIT`](https://github.com/ChainSafe/forest/issues/1384#issuecomment-1100860018)

Unfortunately it seems that `FOREST_VERSION` which is created in the `forest` crate cannot be imported into `forest_libp2p` since `forest` itself depends on `forest_libp2p` first.
As documented at:
[`forest` and `forest_libp2p` circular dependency](https://github.com/ChainSafe/forest/issues/1384#issuecomment-1100860634)